apps/server/src/app.module.ts

```diff
@@ -4,10 +4,12 @@ import { DataSource } from 'typeorm';
 import { TypeOrmModule } from '@nestjs/typeorm';
 import { AppController } from './app.controller';
 import { AppService } from './app.service';
+import { AuthModule } from './auth/auth.module';
+import { UserModule } from './user/user.module';
 import databaseConfig from './config/database.config';
 
 @Module({
-  imports: [ConfigModule.forRoot(), TypeOrmModule.forRoot(databaseConfig())],
+  imports: [ConfigModule.forRoot(), TypeOrmModule.forRoot(databaseConfig()), AuthModule, UserModule],
```

Comment:
Also, we might need to opt into using `TypeOrmModule.forRootAsync()` to inject ConfigService for secret env variables.

Reply:
updated usages to `TypeOrmModule.forRootAsync()`